src/renderer/hooks/useMapServerQueryWithImplicitTypings.js

```diff
@@ -0,0 +1,26 @@
+// @ts-check
+import ky from 'ky/umd'
+import { useQuery } from '@tanstack/react-query'
+
+// local host
+// global port number
+const MAP_SERVER_URL = 'http://127.0.0.1:' + window.mapServerPort
+
+const apiLayer = () => {
+  function getStylesFn () {
+    /**
+     * @type {Promise<ReturnType<import('@mapeo/map-server/dist/api/styles').StylesApi["listStyles"]>>}
+     */
+    const listSyles = ky.get(MAP_SERVER_URL + '/styles').json()
+    return listSyles
+  }
+
+  return {
+    getStyle: getStylesFn
+  }
+}
+
+const { data } = useQuery({
```

Comment:
@gmaclennan with an API layer, the `data` type is implicitly defined by out type assertions in API layer

Reply:
See above for a solution that doesn't require this file.